packages/deck-scripts/helpers/rollup-plugin-angularjs-template-loader.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
```

Comment:
These changes are copied over from `deck-kayenta`. After publishing these changes, I'll make `deck-kayenta` rely on `deck-scripts` and remove one-off customizations.